CMakeLists.txt

```diff
@@ -0,0 +1,684 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
Remove this file from the PR. You could list this file in .gitgnore if that would be helpful to you in the future.